MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+
+### TextAlignment
+
+The `HorizontalTextAlignment ` and `VerticalTextAlignment ` properties are used to customize the alignment of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" HorizontalTextAlignment="Center" VerticalTextAlignment="Center">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.HorizontalTextAlignment = TextAlignment.Center;
+button.VerticalTextAlignment = TextAlignment.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Background Customization
+
+The background of the button can be customized by its background color, border color, border width and corner radius.
+
+### BackgroundColor
+
+The `Background` property is used to customize the background color of `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" Background = "DeepSkyBlue">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.Background = Colors.DeepSkyBlue;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with background color](images/Button_backgroundcolor.png)
+
+### Stroke
+
+The `BorderColor` property is used to customize the color of border in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" Stroke="Red">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.Stroke = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with stroke](images/Button_border.png)
+
+### StrokeThickness
+
+The `BorderWidth` property is used to customize the thickness of border in `SfButton`. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" Stroke="Red" StrokeThickness="6">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.StrokeThickness = 6;
+button.Stroke = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with strokethickness](images/Button_borderthickness.png)
+
+### CornerRadius
+
+The `CornerRadius` property is used to customize the rounded edges in `SfButton` as demonstrated in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" CornerRadius="20">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.CornerRadius = 20;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with cornerradius](images/Button_cornerradius.png)
+
+## Image Customization
+
+The Image can be customized by its ShowIcon, ImageSource, ImageSize and ImageAlignment.
+
+### ShowIcon
+
+You can enable the Icon image using the `ShowIcon` property to know whether any image appears to the `SfButton`.
```

Comment:
You can enable the Icon image using the `ShowIcon` property to know whether any image appears in the `SfButton`.